src/server/server.py

```diff
@@ -5433,6 +5488,21 @@ def determineHostAddress(maxRetrySecs=10):
 if vrx_controller:
     Events.on(Evt.CLUSTER_JOIN, 'VRx', killVRxController)
 
+# Search for data export plugins
+exporters = {}
+export_plugins = Plugins(prefix='data_export')
+export_plugins.discover()
+# collect exporters into single list
+for exporter in export_plugins:
+    if exporter['id'] in exporters:
+        logger.warning('Overwriting data exporter "{0}"'.format(exporter['id']))
+
+    exporters[exporter['id']] = {
+        'name': exporter['name'],
+        'handlerFn': exporter['handlerFn'],
+        'args': exporter['args'],
```

Comment:
This seems like unnecessary copying.

Reply:
Probably is. What's the alternative? I wasn't sure how to access them by ID otherwise.

Reply:
Exporters[exporter[id]] =exporter
More fundamental question, do we need both a name and an id?

Reply:
I wasn't sure it was a good idea to have an ID with spaces, punctuation, etc. that will show up in the UI. If we can be certain that it's safe, then there's no need.